directory.html

```diff
@@ -6,98 +6,107 @@
     <meta name="theme-color" content="#000000">
     <link href="https://fonts.googleapis.com/css?family=Lato:100,200,300,400" rel="stylesheet">
 
-
-    <title>Directory</title>
-
+    <title>CCA MFA in Comics</title>
 
     <link href="index.css" rel="stylesheet"/>
-
-
-
+    <link rel="shortcut icon" type="image/png" href="images/ui/favicon.png" />
 
 </head>
 
-
-
+<!--
+TODO
+[x] add images to cards
+[x] figure out better way to delinate by year
+[ ] why doesn't 2016 work
+[ ] do something about people with no digital
+-->
 
 <body>
 
 
-
     <noscript>
         You need to enable JavaScript to run this app.
     </noscript>
 
-
+<div class="transp">
```

Comment:
I recommend spelling the word out ("transparent"?).  Clarity is much more important than saving a few bytes.

Reply:
👍🏻